primitives/bridge/src/lib.rs

```diff
@@ -374,6 +377,9 @@ where
             (1, [GlobalConsensus(Ethereum { chain_id })]) => {
                 Some(Self::from_chain_id(chain_id).into())
             }
+            (2, [GlobalConsensus(Ethereum { chain_id })]) => {
```

Comment:
while I do understand that you want to reduce the code, this should not belong here. it should be a separate struct. it is a bad idea to join the ethereum converter for a container chain with the assumptions of the solo-chain orchestrator.